test/test_format.py

```diff
@@ -12,6 +12,8 @@
 @helpers.filtered_test
 def test_format(implementation: pqclean.Implementation):
     helpers.ensure_available('astyle')
+    result = helpers.run_subprocess(['astyle', '--version'])
+    print(result)
```

Comment:
I don't think we intend to commit this

Reply:
I do, actually; different astyle versions lead to different formatting, so it can be useful to have that information. Unless it causes a problem elsewhere?

Reply:
Although it seems like astyle is fighting with itself on different platforms. I'm not sure how much we want to get that resolved?

Reply:
I agree with @dstebila , my previous code passed all astyle checks, but this time astyle test failed in the non-commit code.